ansible/roles/screenly/files/screenly_utils.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+recover(){
+	sudo service screenly-viewer stop
+	rm -rf /home/pi/.screenly /home/pi/screenly_assets
+	tar -xvf "$1" -C /home/pi/
```

Comment:
Before starting the recovery, we need to:
 * Verify that $1 is a string
 * The file exists